lib/roo_on_rails/checks/git/origin.rb

```diff
@@ -0,0 +1,24 @@
+require 'roo_on_rails/checks/base'
+
+module RooOnRails
+  module Checks
+    module Git
+      class Origin < Base
+        def _intro
+          "Checking your Git origin remote..."
+        end
+
+        def _call
+          status, url = _run "git config remote.origin.url"
+          _fail "Origin does not seem to be configured." unless status
+          
+          org, repo = url.strip.gsub(%r{\.git$}, '').split(%r{[:/]}).last(2)
```

Comment:
If `git config remote.origin.url` doesn't return a single-line URL we have bigger problems. I think using `\Z` everywhere is a bit pedantic when the input is well-known ;)
(point taken on `sub`, will fix)